### ISSUE TITLE
Unable to run Hello world example

### ISSUE THREAD

**Issue by Zumbalamambo** [google/mediapipe#1035]
I'm getting the following error while trying to run the hello world example.

```
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
INFO: Options provided by the client:
  Inherited 'common' options: --isatty=1 --terminal_columns=80
INFO: Reading rc options for 'run' from /home/Tools/mediapipe/.bazelrc:
  Inherited 'common' options: --experimental_repo_remote_exec
ERROR: Unrecognized option: --experimental_repo_remote_exec
INFO: Writing tracer profile to '/root/.cache/bazel/_bazel_root/1c701555ce36f7c7611f6677aac9bb84/command.profile.gz'

```

**Comment by jiuqiant** [comment]
What's your bazel version? Also, https://github.com/google/mediapipe/issues/752 might be helpful.

**Comment by dgrnd4** [comment]
@Zumbalamambo  is my same problem at #1018 ?

**Comment by jiuqiant** [comment]
@Zumbalamambo, have you solved the problem by switching to v0.7.5 with Bazel 2.0.0+？

**Comment by jiuqiant** [comment]
We are closing this issue for now due to lack of activity.
